apps/remote_control/lib/lexical/remote_control/build/error_support.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Lexical.RemoteControl.Build.ErrorSupport do
+  def context_to_position(context) do
+    cond do
+      Keyword.has_key?(context, :line) and Keyword.has_key?(context, :column) ->
```

Comment:
Pull out line and column as a tuple, then pattern match in the function heads. I think that's be cleaner.

Reply:
```elixir
case {context[:line], context[:column], context[:end_line], context[:end_collumn]} do 
  {nil, nil, nil, nil} -> nil # I don't like returning nil here. This leaves us without a place to put the error
  {line, nil, nil, nil} -> line
  {line, column, nil, nil} -> {line, column}
  range -> range
end
```

I kind of wonder if we should just be returning Positions and Ranges here.